gems/aws-sdk-s3/lib/aws-sdk-s3/plugins/s3_signer.rb

```diff
@@ -67,7 +64,6 @@ def sigv4_signer(context)
             if
               context[:cached_sigv4_region] &&
               context[:cached_sigv4_region] != context.config.sigv4_signer.region
-            then
```

Comment:
If you are removing `then` (old syntax) then please align the `if` statement above.